frontend/components/site/siteBuilds.jsx

```diff
@@ -280,19 +320,30 @@ function SiteBuilds() {
                         </div>
                         <span>
                           {
-                          ['error', 'success'].includes(build.state)
-                          && (
-                          <CreateBuildLink
-                            handlerParams={{ buildId: build.id, siteId: site.id }}
-                            handleClick={buildActions.restartBuild}
-                            className="usa-button small-button rebuild-button"
-                          >
-                            Rebuild
-                          </CreateBuildLink>
-                          )
-                        }
+                            ['error', 'success'].includes(build.state)
+                            && (
+                              <CreateBuildLink
+                                handlerParams={{ buildId: build.id, siteId: site.id }}
+                                handleClick={buildActions.restartBuild}
+                                className="usa-button small-button rebuild-button"
+                              >
+                                Rebuild
+                              </CreateBuildLink>
+                            )
+                          }
                         </span>
                       </td>
+                      { process.env.FEATURE_BUILD_TASKS === 'active' && (
+                        hasBuildTasks(build)
+                          ? (
+                            <td data-title="Scans" className="scan-results-pseudo-row">
+                              { summarizeTaskResults(build) }
+                            </td>
+                          )
+                          : (
+                            <td className="no-scan-results usa-sr-only">No scans for this build</td>
```

Comment:
An assortment of thoughts:
- Screenreader and visual understanding should match as much as we can
- Totally independent of whether this feature is active, users without any build scans shouldn't have them displayed/mentioned (although maybe this is only true during the beta)
- One potential solution is not treating the scan portion as a `<td>` (goes back to our card vs table row discussion) which avoids the need to set a column header
- A quick proxy for whether the site has any tasks (across all builds) is whether `Site.SiteBuildTasks` has any results but we don't currently return that from the API
- Also it's theoretically very quick to check if the displayed builds have tasks (something like `builds.some(build => build.BuildTasks.length)` and it will correctly be true per page)